plugins/inputs/lotus/lotus.go

```diff
@@ -296,13 +335,38 @@ func recordTipsetStatePoints(ctx context.Context, api api.FullNode, acc telegraf
 	return nil
 }
 
+func recordMpoolUpdatePoints(ctx context.Context, acc telegraf.Accumulator, newMpoolUpdate api.MpoolUpdate, receivedAt time.Time, receivedFrom peer.ID) error {
+	var updateType string
+	switch newMpoolUpdate.Type {
+	case api.MpoolAdd:
+		updateType = "MpoolAdd"
+	case api.MpoolRemove:
+		updateType = "MpoolRemove"
+	default:
+		return fmt.Errorf("unknown mpool update type: %v", newMpoolUpdate.Type)
+	}
+
+	acc.AddFields("chain_mpool",
+		map[string]interface{}{
+			"message_size": newMpoolUpdate.Message.Size(),
+			"recorded_at":  receivedAt.UnixNano(),
+		},
+		map[string]string{
+			"message_cid_tag":       newMpoolUpdate.Message.Cid().String(),
+			"mpool_update_type_tag": updateType,
+			"recorded_from":         receivedFrom.String(),
```

Comment:
Depending on what we decide for `recievedFrom` we might use that same term here. (Whatever label we use should work in this context as well.)